test/new_project.sh

```diff
@@ -8,3 +8,8 @@ npm install
 npm uninstall near-shell
 npm install ../
 NODE_ENV=development npm run test
+timestamp=$(date +%s)
```

Comment:
let's put this into separate script, e.g. `test/create_account.sh`?

Reply:
or maybe have a setup script which sets up new project and then run few different scenarios in separate scripts after that:

- run tests
- create account
- deploy